src/middlewares/errorHandler.js

```diff
@@ -0,0 +1,22 @@
+import Debug from 'debug';
+import { config } from 'dotenv';
+
+config();
+const debug = Debug('dev');
+export default (err, request, response, next) => {
+  const isProduction = process.env.NODE_ENV === 'production';
+
+  let errorMessage = {};
+  if (!isProduction) {
+    debug(err.stack);
+    errorMessage = err;
+  }
+  response.status(err.status || 500);
```

Comment:
Please leave a new line between lines 13 and 14

Reply:
Also kindly chain .status() and .json() and return the response. We want to avoid getting `headers already sent` errors.

```suggestion
  return response
    .status(err.status || 500)
    .json({
       errors: {
         message: err.message,
         error: errorMessage
       }
     });
```